components/Questionnaire/index.vue

```diff
@@ -157,12 +162,30 @@ export default {
         this.onSubmit()
       }
     },
+    onNextLevelThree (properties) {
+      this.params.properties.tentang_bumdes = properties.tentang_bumdes
+      this.params.properties.potensi_desa = properties.tentang_bumdes
```

Comment:
why assign the same value? maybe another one should be `potensi_desa` ?

Reply:
I think it's wrong kak, you can see the changes [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/94/commits/e0d3665516e6f4466539a34050520058f19dcfab)